### ISSUE TITLE
key error DEBUG in Flask?

### ISSUE THREAD

**Issue by id2359** [GibbsConsulting/django-plotly-dash#425]
Hello, I am seeing the following error:
```




File "/app/rdrf/dashboards/views.py", line 101, in get
return render(request, "rdrf_cdes/patients_dashboard.html", context)

I just instantiate a test Django-Plotly-Dash app, but I'm running into the following:





File "/env/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
content = loader.render_to_string(template_name, context, request, using=using)
File "/env/lib/python3.8/site-packages/django/template/loader.py", line 62, in render_to_string
return template.render(context, request)
File "/env/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
return self.template.render(context)
File "/env/lib/python3.8/site-packages/django/template/base.py", line 170, in render

return self._render(context)
File "/env/lib/python3.8/site-packages/django/template/base.py", line 162, in _render
return self.nodelist.render(context)
File "/env/lib/python3.8/site-packages/django/template/base.py", line 938, in render
bit = node.render_annotated(context)
File "/env/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
return self.render(context)
File "/env/lib/python3.8/site-packages/django/template/library.py", line 214, in render
_dict = self.func(*resolved_args, **resolved_kwargs)
File "/env/lib/python3.8/site-packages/django_plotly_dash/templatetags/plotly_dash.py", line 76, in plotly_app
da, app = _locate_daapp(name, slug, da, cache_id=cache_id)
File "/env/lib/python3.8/site-packages/django_plotly_dash/templatetags/plotly_dash.py", line 43, in _locate_daapp
da, app = DashApp.locate_item(name, stateless=True, cache_id=cache_id)
File "/env/lib/python3.8/site-packages/django_plotly_dash/models.py", line 231, in locate_item
app = dash_app.as_dash_instance(cache_id=cache_id)
File "/env/lib/python3.8/site-packages/django_plotly_dash/dash_wrapper.py", line 227, in as_dash_instance
return self.do_form_dash_instance(cache_id=cache_id)
File "/env/lib/python3.8/site-packages/django_plotly_dash/dash_wrapper.py", line 263, in do_form_dash_instance
return self.form_dash_instance(replacements, ndid, base_pathname)
File "/env/lib/python3.8/site-packages/django_plotly_dash/dash_wrapper.py", line 271, in form_dash_instance
rd = WrappedDash(base_pathname=base_pathname,
File "/env/lib/python3.8/site-packages/django_plotly_dash/dash_wrapper.py", line 429, in __init__
super().__init__(__name__, **kwargs)
File "/env/lib/python3.8/site-packages/dash/dash.py", line 494, in __init__
self.init_app()
File "/env/lib/python3.8/site-packages/dash/dash.py", line 558, in init_app
self.server.before_request(self._setup_server)
File "/env/lib/python3.8/site-packages/flask/scaffold.py", line 39, in wrapper_func
if self._is_setup_finished():
File "/env/lib/python3.8/site-packages/flask/app.py", line 517, in _is_setup_finished
return self.debug and self._got_first_request
File "/env/lib/python3.8/site-packages/flask/app.py", line 792, in debug
return self.config["DEBUG"]
KeyError: 'DEBUG'
```

Something has changed I think, as this error wasn't appearing ( to me) before.

**Comment by mfriedy** [comment]
Seems to be a new incompatability to dash 2.7.0 which wasnt present in 2.6.2

**Comment by id2359** [comment]
Thanks , I'll try pinning. In general, is there a way to pass config information ( provide the DEBUG key ) to Flask via Django-Plotly-Dash?

**Comment by id2359** [comment]
That worked - thanks @mfriedy !

**Comment by GibbsConsulting** [comment]
Should be fixed by PR #429 which has been released as v2.1.3 - please reopen or create a new issue if not resolved
